BedrockCore.cpp

```diff
@@ -44,8 +44,7 @@ uint64_t BedrockCore::_getRemainingTime(const unique_ptr<BedrockCommand>& comman
 
     // Already expired.
     if (adjustedTimeout <= 0 || (isProcessing && processTimeout <= 0)) {
-        SALERT("Command " << command->request.methodLine << " timed out after "
-               << ((now - command->request.calc64("commandExecuteTime")) / 1000) << "ms.");
+        SALERT("Command " << command->request.methodLine << " timed out.");
```

Comment:
Would it be worth showing the timeout time if `commandExecuteTime` is set? Seems like potentially valuable information

Reply:
Discussed with Tyler, sounds like the log line is only needed if we are diagnosing performance issues, which is not the case currently.